resources/dicts/thoughts/alive/medicine_cat.json

```diff
@@ -73,6 +118,18 @@
             "calm"
         ]
     },
+    {
+        "id": "calm_medicine_cat_apprentice",
+        "thoughts": [
+            "Calms down an apprentice who is panicking other their minor injury"
```

Comment:
shld be 'over their minor injury'

Reply:
"their" also needs to be pronoun tagged

Reply:
ah I thought I should leave it ambiguous as to which apprentice the thought refers to, but i'll change it to be pronoun tagged